staff/jaime/camel-case-to-lower-case-with-spaces/scripts/main.js

```diff
@@ -0,0 +1,7 @@
+function camelCase(text) {
+    var regEx = /[A-Z\d]+/g;
+    text = text.replace(regEx, function (n) { return ' ' + n; });
+    return text;
```

Comment:
You can return the result of text.replace like `return text.replace(regEx, function (n) { return ' ' + n; });` and we have the same result... by the way, good function.

+1